server/tests/client/federated.spec.ts

```diff
@@ -48,16 +49,19 @@ describe("federated client", () => {
           validationSplit: 0,
           batchSize: 1,
           scheme: "federated",
+          minNbOfParticipants: 2,
           dataType: "tabular",
           tensorBackend: 'tfjs'
         },
       },
       aggregators.getAggregator(TASK),
     );
 
+    let model
     try {
-      await client.connect();
+      model = await client.connect();
     } catch {
+      expect(model).equal(undefined)
```

Comment:
undef is soo common, chai added [a special way to check it](https://www.chaijs.com/api/bdd/#method_undefined)

```suggestion
      expect(model).to.be.undefined
```